shell/components/PromptRemove.vue

```diff
@@ -280,11 +280,17 @@ export default {
     },
     async parallelRemove(btnCB) {
       try {
+        if (typeof this.toRemove[0]?.bulkRemove === 'undefined') {
+          await Promise.all(this.toRemove.map(resource => resource.remove()));
+          this.done();
```

Comment:
these `this.done` should remain in the original place, and `this.toRemove[0]?.bulkRemove` should be await